widgets/src/main/java/org/pentaho/mantle/client/dialogs/scheduling/AdditionalDetailsPanel.java

```diff
@@ -0,0 +1,117 @@
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details.
+ *
+ * Copyright (c) 2023-2024 Hitachi Vantara..  All rights reserved.
+ */
+
+package org.pentaho.mantle.client.dialogs.scheduling;
+
+import com.google.gwt.user.client.ui.Composite;
+import com.google.gwt.user.client.ui.ListBox;
+import com.google.gwt.user.client.ui.CheckBox;
+import com.google.gwt.user.client.ui.CaptionPanel;
+import com.google.gwt.user.client.ui.Label;
+import org.pentaho.gwt.widgets.client.controls.ErrorLabel;
+import org.pentaho.gwt.widgets.client.panel.HorizontalFlexPanel;
+import org.pentaho.gwt.widgets.client.panel.VerticalFlexPanel;
+import org.pentaho.gwt.widgets.client.ui.ICallback;
+import org.pentaho.gwt.widgets.client.ui.IChangeHandler;
+import org.pentaho.gwt.widgets.client.utils.TimeUtil;
+import org.pentaho.mantle.client.messages.Messages;
+
+public class AdditionalDetailsPanel extends Composite  implements IChangeHandler {
+
+    private ListBox logLevelLb = createLogLevelListBox();
+    private ICallback<IChangeHandler> onChangeHandler = null;
+    private ErrorLabel detailsLabel = null;
+    private CheckBox enableSafeMode;
+    private CheckBox gatherMetrics;
+
+    private static final String SCHEDULER_ADDITIONAL_PANEL = "schedule-editor-additional-panel"; //$NON-NLS-1$
+    private static final String SCHEDULE_DETAILS_INPUT = "schedule-details-input";
+
+    public AdditionalDetailsPanel() {
+        CaptionPanel detailPanel = new CaptionPanel( Messages.getString( "schedule.detail" ) );
+        detailPanel.setStyleName( SCHEDULER_ADDITIONAL_PANEL );
+        detailPanel.getElement().setId( SCHEDULE_DETAILS_INPUT );
+
+        VerticalFlexPanel dVP = new VerticalFlexPanel();
+        enableSafeMode = new CheckBox();
+        enableSafeMode.setText( Messages.getString( "schedule.enableSafeMode" ) ); //$NON-NLS-1$
+        enableSafeMode.setStyleName( "enableSafeModeCheckbox" ); //$NON-NLS-1$
+        dVP.add( enableSafeMode );
+
+        gatherMetrics = new CheckBox();
+        gatherMetrics.setText( Messages.getString( "schedule.performanceMetrics" ) ); //$NON-NLS-1$
+        gatherMetrics.setStyleName( "gatherPerfmnceMetricsCheckbox" ); //$NON-NLS-1$
+        dVP.add( gatherMetrics );
+
+        HorizontalFlexPanel logLvlHP = new HorizontalFlexPanel();
+        logLevelLb.getElement().setId( "log-level-lb" );
+        logLvlHP.add( logLevelLb );
+        Label l = new Label( Messages.getString( "schedule.logLevel" ) );
+        l.setStyleName( "endLabel" ); //$NON-NLS-1$
+        logLvlHP.add( l );
+        dVP.add( logLvlHP );
+        detailPanel.add( dVP );
+
+        detailsLabel = new ErrorLabel( detailPanel );
+        VerticalFlexPanel detailVP = new VerticalFlexPanel();
+        detailVP.add( detailsLabel );
```

Comment:
Please, be consistent in naming. Unless there is some reason, these should all either use the `details` or the `detail` prefix (I believe the former is more common).